doc/articles/features/working-with-accessibility.md

```diff
@@ -90,13 +90,10 @@ When the value of `WinRTFeatureConfiguration.Accessibility.HighContrast` is chan
 
 ## Known issues
 
-- `Hyperlink` in `TextBlock` is not supported.
-- `TextBox` and `PasswordBox` don't use `Header` and `PlaceholderText`.
```

Comment:
Same here, I think on a11y focus it's expected that screen readers will read the header and placeholder.